src/main/java/spoon/reflect/code/CtAbstractInvocation.java

```diff
@@ -49,6 +49,12 @@
 	<C extends CtAbstractInvocation<T>> C addArgument(CtExpression<?> argument);
 
 	/**
+	 * Adds an argument expression to the invocation at the specified position
+	 */
```

Comment:
Shifts the element currently at that position (if any) and any subsequent elements to the right (adds one to their indices).